src/Modules/UngradedModules/SubModuleOfFreeModule.jl

```diff
@@ -164,6 +164,7 @@ Compute a standard basis of `submod` with respect to the given `odering``.
 The return type is `ModuleGens`.
 """
 function standard_basis(submod::SubModuleOfFreeModule; ordering::ModuleOrdering = default_ordering(submod))
+  @req is_exact_type(typeof(base_ring(submod)(0))) "This functionality is only supported over exact fields."
```

Comment:
```suggestion
  @req is_exact_type(elem_type(base_ring(submod))) "This functionality is only supported over exact fields."
```
this avoids constructing an element. can be applied to most other places as well

Reply:
Thanks, adjusted correspondingly.